docs/extendedjob.md

```diff
@@ -1,8 +1,8 @@
-# ExtendedJob
+# QuarksJob
```

Comment:
Does the document itself should change the name?
from `extendedjob.md` to `quarksjob.md` I think